docs/api/update.md

```diff
@@ -0,0 +1,293 @@
+|  3   | INVALID_ARGUMENT |
+|  6   | ALREADY_EXISTS   |
+|  13  | INTERNAL         |
+
+## StreamUpdate RPC<sup>recommended</sup>
+
+StreamUpdate RPC is the method to add new multiple vectors using the [bidirectional streaming RPC](https://grpc.io/docs/what-is-grpc/core-concepts/#bidirectional-streaming-rpc).<br>
+By using the bidirectional streaming PRC, the update request can be communicated in any order between client and server.
+Each Update request and response are independent.
+It's the recommended method to update the large amount of vectors.
+
+### Input
+
+- the scheme of `payload.v1.Update.Request stream`
+
+  ```bash
+  message Update {
+      message Request {
+          Object.Vector vector = 1 [ (validate.rules).repeated .min_items = 2 ];
+          Config config = 2;
+      }
+      message Config {
+          bool skip_strict_exist_check = 1;
+          Filter.Config filters = 2;
+          int64 timestamp = 3;
+      }
+  }
+
+  message Object {
+      message Vector {
+          string id = 1 [ (validate.rules).string.min_len = 1 ];
+          repeated float vector = 2 [ (validate.rules).repeated .min_items = 2 ];
+      }
+  }
+  ```
+
+  - Update.Request
+    |field|type|label|required|desc.|
+    |:---:|:---|:---|:---:|:---|
+    |vector|Object.Vector| | \* | the information of vector |
+    |config|Config| | \* | the configuration of the update request |
+
+  - Update.Config
+    |field|type|label|required|desc.|
+    |:---:|:---|:---|:---:|:---|
+    |skip_strict_exist_check|bool| | | check the same vector is already inserted or not.<br>the ID should be unique if the value is `true`|
+    |timestamp|int64| | | the timestamp of the vector updated.<br>if it is N/A, the current time will be used.
+    |filters|Filter.Config| | | configuration for filter |
+
+  - Object.Vector
+    |field|type|label|required|desc.|
+    |:---:|:---|:---|:---:|:---|
+    |id|string| | \* | the ID of the vector. ID should consist of 1 or more characters. |
+    |vector|float| repeated(Array[float]) | \* | the vector data. its dimension is between 2 and 65,536.|
+
+### Output
+
+- the scheme of `payload.v1.Object.StreamLocation`
+
+  ```bash
+  message Object {
+      message StreamLocation {
+        oneof payload {
+            Location location = 1;
+            google.rpc.Status status = 2;
+        }
+      }
+
+      message Location {
+        string name = 1;
+        string uuid = 2;
+        repeated string ips = 3;
+      }
+  }
+  ```
+
+  - Object.StreamLocation
+    |field|type|label|desc.|
+    |:---:|:---|:---|:---|
+    |location|Object.Location| | the information of Object.Location data. |
+    |status|google.rpc.Status| | the status of google RPC. |
+
+  - Object.Location
+    |field|type|label|desc.|
+    |:---:|:---|:---|:---|
+    |name|string| | the name of vald agent pod where the request vector is inserted. |
+    |uuid|string| | the ID of the inserted vector. It is the same as an Object.Vector |
```

Comment:
```suggestion
    |uuid|string| | the ID of the updated vector. It is the same as an Object.Vector |
```

Reply:
umm, `already inserted vector` is correct?

Reply:
I think `exist` might be better.
```suggestion
    |uuid|string| | the ID of the exists vector. It is the same as an Object.Vector |
```